spec/ParseUser.spec.js

```diff
@@ -2300,6 +2300,66 @@ describe('Parse.User testing', () => {
     });
   });
 
+  it('case insensitive login with username', async () => {
+    let user = new Parse.User();
+    user.set('username', 'test1');
+    user.set('password', 'test');
+    await user.signUp();
+    await Parse.User.logOut();
+    user = new Parse.User();
+    user.set('username', 'TEST1');
+    user.set('password', 'test');
+    await user.logIn();
+  });
+
+  it('case insensitive login with username should fail with wrong password', async done => {
+    let user = new Parse.User();
+    user.set('username', 'test1');
+    user.set('password', 'test');
+    await user.signUp();
+    await Parse.User.logOut();
+    user = new Parse.User();
+    user.set('username', 'TEST1');
+    user.set('password', 'test1');
+    try {
```

Comment:
Use expectAsync instead of try/catch, also all others below.

Reply:
Changed to expectAsync.